gc/base/Configuration.cpp

```diff
@@ -162,41 +143,51 @@ MM_EnvironmentBase*
 MM_Configuration::createEnvironment(MM_GCExtensionsBase* extensions, OMR_VMThread* omrVMThread)
 {
 	MM_EnvironmentBase* env = allocateNewEnvironment(extensions, omrVMThread);
-	if (NULL == env) {
-		return NULL;
-	}
-	if (!initializeEnvironment(env)) {
-		env->kill();
-		return NULL;
+	if (NULL != env) {
+		if (initializeEnvironment(env)) {
+			if (extensions->fvtest_disableInlineAllocation) {
```

Comment:
It is not clear where this change came from as it did not exist in this function before. It feels like it belongs in the initializeEnvironment method.